HW03/Vladislav_Mitin/spec/task_2_spec.rb

```diff
@@ -0,0 +1,45 @@
+require_relative '../vladislav_mitin_hw_03_t_02'
+
+describe 'task2' do
```

Comment:
```suggestion
RSpec.describe 'task2' do
  subject { task1(logs) }
```

Same comments as the first spec